user/api.go

```diff
@@ -436,6 +437,7 @@ func (a *Api) CreateUser(res http.ResponseWriter, req *http.Request) {
 			a.logAudit(req, &tokenData, "CreateUser isClinic{%t}", newUser.IsClinic())
 			res.Header().Set(TP_SESSION_TOKEN, sessionToken.ID)
 			a.sendUserWithStatus(res, newUser, http.StatusCreated, false)
+			a.logger.Infof("creation request succedeed for username: %s", *newUserDetails.Username)
```

Comment:
```suggestion
                        a.logger.Infof("creation request succedeed for username: %s", *newUserDetails.Username)
			a.logger.Tracef("user payload: %s", *newUserDetails)
```
This would help to understand what is received/decoded by shoreline, especially the "sanitized" strings.